### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by Methuselah96**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,59 +25,55 @@
...
     "ts-node": "^7.0.1",
-    "tslint": "^5.11.0",
+    "tslint": "^5.12.1",
     "tslint-loader": "^3.6.0",
-    "typescript": "^3.1.3",
-    "webpack": "^4.21.0",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.9"
-  },
-  "peerDependencies": {
```
Comment (toxicFork) [inline package.json]: Looks like the peers got reverted here

**Comment by Methuselah96** [comment]
@toxicFork Fixed. Merge conflict issues. :)

File: package.json (hunk 1 above)
Comment (toxicFork) [reply package.json]: I spoke too soon :p

**Comment by toxicFork** [comment]
All merged, thanks for the PRs! I'll be able to hit the publish button when I get home :)

**Comment by toxicFork** [comment]
0.0.6 is now out :)

**Pull request merged**